PrimeNumbers.java

```diff
@@ -16,17 +16,61 @@ public static void myassert(boolean  x) {
 
     private static int secondMaxSixDigitPrime(){
         //WRITE YOUR CODE HERE
+    	int i = 1000000;    	
+    	int check = 0;
+    	while(i>1){
+    		if(isPrime(i)){
+    			check++;
+    		}
+    		if(check==2){
+    			return i;
+    		}
+    		i--;
+    		
+    	}
+			return i;
+		
     }
 
+    private static boolean isPrime(int primeNum){
+    	boolean isPrime = false;
+    	int count=0;
+    	if(primeNum%2!=0){	
 
+			for(int j = 3;j<=primeNum;){
```

Comment:
Do we need to go till primeNum, how about just only to primeNum/2 or in the best case just sqrt(primeNum) will do instead of j<=primeNum